hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/MockNodeManager.java

```diff
@@ -275,12 +278,15 @@ public List<DatanodeDetails> getNodes(NodeStatus status) {
   @Override
   public List<DatanodeDetails> getNodes(
       HddsProtos.NodeOperationalState opState, HddsProtos.NodeState nodestate) {
+    Instant initialInstant = Instant.now();
+    ZoneId zoneId = ZoneId.systemDefault();
+    TestClock testClock = new TestClock(initialInstant, zoneId);
```

Comment:
It would be nice to avoid duplicating these.  Can you please try using `TestClock.newInstance()`?